hathor/conf/testnet.yml

```diff
@@ -54,11 +54,11 @@ FEATURE_ACTIVATION:
     # NOP feature to test Feature Activation for Transactions
     NOP_FEATURE_1:
       bit: 0
-      # N = 4_354_560
-      # Expected to be reached around Sunday, 2024-11-17.
-      # Right now the best block is 4_326_600 on testnet (2024-11-07).
-      start_height: 4_354_560  # N
-      timeout_height: 4_394_880  # N + 2 * 20160 (2 weeks after the start)
+      # N = 4_394_880
+      # start_height expected to be reached around Sunday, 2024-12-01.
+      # Right now the best block is 4_377_375 on testnet (2024-11-25).
+      start_height: 4_394_880  # N
+      timeout_height: 4_435_200  # N + 2 * 20160 (2 weeks after the start)
```

Comment:
What would be the downside of giving it a much more generous timeout height?

Reply:
No downsides, it's just that since this is testnet and we control mining, the feature should be accepted in the first week. But I just changed it to 4 weeks.